Templates/AppProject.stencil

```diff
@@ -1,6 +1,43 @@
 import ProjectDescription
 import ProjectDescriptionHelpers
 
-let project = Project.app(name: "{{ name }}", platform: .{{ platform }}, dependencies: [
-    .project(target: "{{ name }}Kit", path: .relativeToManifest("../{{ name }}Kit"))
-])
+/*
+                +-------------+
+                |             |
+                |     App     | Contains {{ name }} App target and {{ name }} unit-test target
+                |             |
+         +------+-------------+-------+
+         |         depends on         |
+         |                            |
+ +----v-----+                   +-----v-----+
+ |          |                   |           |
+ |   Kit    |                   |     UI    |   Two independent frameworks to share code and start modularising your app
+ |          |                   |           |
+ +----------+                   +-----------+
+
```

Comment:
Sorry I'm late to the game but I"m wondering if we shouldn't also provide unit test targets for `Kit` and/or `UI`?

Reply:
I mention in the description that:

```I also deleted test targets for kit and ui- to keep it as simple as possible. I think it's pretty obvious that you can extend the project with other tests targets, since there is a test target for the app.```

wdyt?

Reply:
Sorry I should have read that more properly! Sounds good 👍